### DISCUSSION TITLE
Does Drois have a function similar to the MySQL INTERVAL() function

### DISCUSSION THREAD

**Discussion by xavierzhao** [apache/doris#21656] [A - General / Q&A] [answered]
MySQL `INTERVAL()` function returns the index of the argument that is more than the first argument.

Syntax:

INTERVAL(N,N1,N2,N3,...)

Example: MySQL INTERVAL() function

The following MySQL statement the 1st number is 85, which is less than 175. All other numbers in the argument are smaller than 85. So, it returns the index of 175. Notice that indexing is started with the 2nd number and first position is zero.

`SELECT INTERVAL(85, 1, 75, 17, 30,56, 175);`

```
mysql> SELECT INTERVAL(85, 1, 75, 17, 30,56, 175);

+-------------------------------------+
| INTERVAL(85, 1, 75, 17, 30,56, 175) |
+-------------------------------------+
|                                   5 | 
+-------------------------------------+
1 row in set (0.02 sec)
 ```

@yagagagaga

**Comment by yagagagaga** [comment, marked as answer]
This function is not yet supported. But don't worry, it will soon be supported.
